Demo/Demo Base/DemoDropInViewController.swift

```diff
@@ -144,13 +141,9 @@ class DemoDropInViewController: DemoBaseViewController {
 
     func updatePaymentMethodNonce(_ result: BTDropInResult?) {
         demoView.paymentMethodTypeLabel.isHidden = (result?.paymentMethod == nil)
-        demoView.paymentMethodTypeIcon.isHidden = (result?.paymentMethod == nil)
-        if let nonce = result?.paymentMethod {
-            let paymentMethodType = BTUIKViewUtil.paymentOptionType(forPaymentInfoType: nonce.type)
```

Comment:
We updated the Demo app to no longer use the `BTUIKViewUtil` to fetch the payment method icon view.